### PR TITLE
Condence context menus

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Hold for @kgcreative and Viraj's signoff.

condenced

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by kavilla** [comment]
is the proposal within OSD or are these examples for potentially having the default values in OUI be smaller? so it hits the entire app?

File: src/plugins/data/public/ui/filter_bar/filter_options.tsx [hunk 1]
```diff
@@ -179,13 +179,13 @@ class FilterOptionsUI extends Component<Props, State> {
         panelPaddingSize="none"
         repositionOnScroll
       >
-        <EuiPopoverTitle>
+        <EuiPopoverTitle paddingSize="s">
```
Comment (virajsanghvi) [inline src/plugins/data/public/ui/filter_bar/filter_options.tsx:182]: Do you think its better to do these locally rather than set a default through OUI?

**Review by virajsanghvi — APPROVED** [review]
We probably need to follow up with 1) strategy on which changes will need to be behind a flag, 2) whether popover defaults should be made at OUI level and 3) how to address small context menu titles being a smaller size than popover titles even though body text size is the same, but changes lgtm and these could be addressed after.

File: src/plugins/data/public/ui/filter_bar/filter_options.tsx (hunk 1 above)
Comment (AMoo-Miki) [reply src/plugins/data/public/ui/filter_bar/filter_options.tsx:182]: Due to the potential blast radius, I think we should localize these changes. I am also concerned about different fonts and font-sizes across themes and their impact on all of these.

**Comment by AMoo-Miki** [comment]
> is the proposal within OSD or are these examples for potentially having the default values in OUI be smaller? so it hits the entire app?

There is a big collection of changes. Some are done globally within OUI (via a new theme) and some are done surgically. There will most likely be some that will go behind a config flag too.

**Review by zhongnansu — APPROVED** [review]

**Comment by Hailong-am** [comment]
@AMoo-Miki seem test snapshot update was missed, that cause CI to fail https://github.com/opensearch-project/OpenSearch-Dashboards/actions/runs/9966986613/job/27539949592?pr=7254

**Pull request merged**